scripts/extensions/sams/src/components/assets/assetEditorPanel.tsx

```diff
@@ -55,31 +140,48 @@ class AssetEditorPanelComponent extends React.PureComponent<IProps> {
 
         return (
             <React.Fragment>
+                <PanelHeader borderB={true}>
+                    <PanelHeaderSlidingToolbar>
+                        <ButtonGroup align="right">
+                            <Button
+                                text={gettext('Cancel')}
+                                style="hollow"
+                                onClick={this.onCancel}
+                                disabled={this.state.submitting}
+                            />
+                            <Button
+                                text={gettext('Save')}
+                                type="primary"
+                                disabled={!this.state.isDirty || this.state.submitting}
+                                onClick={this.onSave}
+                            />
+                        </ButtonGroup>
+                    </PanelHeaderSlidingToolbar>
+                </PanelHeader>
                 <FormGroup>
```

Comment:
These `FormGroup`'s are render-able with the `AssetEditor`. Instead of duplicating these render items, it might be better to render `AssetEditor` here instead

Reply:
Resolved.